equistore-core/tests/cpp/external/CMakeLists.txt

```diff
@@ -1,33 +1,3 @@
-function(unpack_library _name_)
```

Comment:
Is the removal related to the cmake update? I just could not function used in the repo

Reply:
yes, the function was never used ^_^ I copy-pasted it from another project thinking I'll need it, but ended up doing things differently